libsolidity/interface/OptimiserSettings.h

```diff
@@ -72,6 +77,8 @@ struct OptimiserSettings
 		OptimiserSettings s = none();
 		s.runJumpdestRemover = true;
 		s.runPeephole = true;
+		s.runYulOptimiser = true;
```

Comment:
Now `runYulOptimizer` is used pretty much only in tests, and even then we could switch to "always enabled" if we really wanted to. I that's the case, I can remove it in a follow-up refactor PR.